js/src/structured-data-blocks/how-to/components/HowTo.js

```diff
@@ -306,28 +306,50 @@ export default class HowTo extends Component {
 
 		return (
 			<div className="schema-how-to-duration">
```

Comment:
Can we remove this div in favour of the fieldset? And move the `className` to the fieldset.